pkgs/by-name/sp/spec-kit/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "spec-kit";
```

```suggestion
  pname = "specify";
```
to align with other package managers: https://repology.org/project/specify/versions. Don't forget to move the file.